src/tls/Ticket.fst

```diff
@@ -59,8 +59,8 @@ private let dummy_id (a:aeadAlg) : St AE.id =
 private let ticket_enc : reference ticket_key
   =
   let id0 = dummy_id CC.CHACHA20_POLY1305 in
-  let salt : AE.salt id0 = CC.zero (AE.iv_length id0) in
-  let key : AE.key id0 = CC.zero (AE.key_length id0) in
+  let salt : AE.salt id0 = CC.random (AE.iv_length id0) in
+  let key : AE.key id0 = CC.random (AE.key_length id0) in
```

Comment:
Top-level effects.
I thought you had fixed this at the FFI level by calling `set_ticket_key` to set a the ticket encryption key externally.
This generates code in `kremlininit_globals` that calls the RNG, so the RNG needs to be initialized before, but there is no good place to do it reliably in the F* sources.